techlibs/xilinx/synth_xilinx.cc

```diff
@@ -187,14 +204,18 @@ struct SynthXilinxPass : public Pass
 
 		if (check_label(active, run_from, run_to, "begin"))
 		{
-			if (vpr)
-			{
-				Pass::call(design, "read_verilog -lib -D_EXPLICIT_CARRY -D_EXPLICIT_DRAM +/xilinx/cells_sim.v");
-			} else {
-				Pass::call(design, "read_verilog -lib +/xilinx/cells_sim.v");
-			}
+            if (vpr) {
+    			Pass::call(design, "read_verilog -lib -D_EXPLICIT_CARRY +/xilinx/cells_sim.v");
+            } else {
+    			Pass::call(design, "read_verilog -lib +/xilinx/cells_sim.v");
+            }
+
 			Pass::call(design, "read_verilog -lib +/xilinx/cells_xtra.v");
-			Pass::call(design, "read_verilog -lib +/xilinx/brams_bb.v");
+
+            if (!noBrams) {
+    			Pass::call(design, "read_verilog -lib +/xilinx/brams_bb.v");
+            }
```

Comment:
This file uses tabs, please be consistent and use tabs?

Reply:
Fixed. (actually tabs were there already...)